doc/index.md

```diff
@@ -42,16 +42,16 @@ calling ``mvn clean package`` will output a WildFly server containing your deplo
 This server has then to be installed into the WildFly runtime image ([JDK11](https://quay.io/repository/wildfly/wildfly-runtime-jdk11) or 
 [JDK 17](https://quay.io/repository/wildfly/wildfly-runtime-jdk17)). 
 
-Use [this dockerfile example](https://github.com/wildfly/wildfly-s2i/blob/main/examples/docker-build/Dockerfile) 
-to build an application image using docker.
+Use [this Dockerfile example](https://github.com/wildfly/wildfly-s2i/blob/main/examples/docker-build/Dockerfile) 
+to build an application image using Docker.
```

Comment:
ok, here we specifically mention the tool so maybe use "Docker or Podman". Or if you have a good definition that encompasses both, that'd be better? "tool to build container image"?

Reply:
While I agree in general, I am on a mission to push `Podman` as the  open source container solution, thus I want to make that explicit.

Reply:
Went for 'Docker or Podman' here.